packages/flame/src/Dropdown/Dropdown.tsx

```diff
@@ -168,6 +169,7 @@ export const Dropdown: React.FC<Props> = ({
             }
           }}
           forcedState={isActive ? 'active' : null}
+          {...(restProps as any)}
```

Comment:
Should we add a test here to prevent future regression? To validate that rest props are properly drilled down the Button.

Reply:
That's like testing every prop for button lol

No, ideally, we should probably not use restprop too often. 
as it stands, its used everywhere. It would probably be a lot safer to simply specify the props instead of doing a spread everytime.

We'll get a tighter API that way and prevent wacky injections via consumers as well.